core/kogitoservice/probe.go

```diff
@@ -21,18 +21,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-// HealthCheckProbeType defines the supported probes for the ServiceDefinition
-type HealthCheckProbeType string
+// ProbeType defines the types of probes supported by K8s
+type ProbeType string
 
 const (
-	// QuarkusHealthCheckProbe probe implemented with Quarkus Microprofile Health. See: https://quarkus.io/guides/microprofile-health.
-	// the operator will set the probe to the default path /health/live and /health/ready for liveness and readiness probes, respectively.
-	QuarkusHealthCheckProbe HealthCheckProbeType = "quarkus"
-	// TCPHealthCheckProbe default health check probe that binds to port 8080
-	TCPHealthCheckProbe HealthCheckProbeType = "TCP"
+	livenessProbeType  ProbeType = "liveness"
+	readinessProbeType ProbeType = "readiness"
+	startupProbeType   ProbeType = "startup"
 
 	quarkusProbeLivenessPath  = "/q/health/live"
```

Comment:
are these paths configurable on the operator?
what happens if the user decides to customize the heath check paths?

Reply:
> are these paths configurable on the operator?

Yes, they are. https://github.com/kiegroup/kogito-operator/blob/2b224716e9b712624c5d58d01d870f582acba2e1/kogito-operator.yaml#L1225-L1227

> what happens if the user decides to customize the heath check paths?

The values here are only used if the user doesn't specify a path in the YAML. Otherwise, their's will be used.

Reply:
cool, thanks man!